web/src/components/Layout.tsx

```diff
@@ -29,14 +29,23 @@ const RichLayout = styled.div`
   }
 `
 
-const Body = styled.div<{ fullWidth: boolean }>`
+const Body = styled.div<{ fullWidth: boolean; disableScrollingSafari: boolean }>`
```

Comment:
hmh scrolling is disabled not only on Safari but on every browser, therefore I think the naming is not really matching :thinking:

Reply:
the style only applies in the styled component for safari in this case. we had a conversation before and i changed it from `disabledScrolling` to be more specific